src/content/en/fundamentals/payments/_toc.yaml

```diff
@@ -1,11 +1,12 @@
 toc:
-- title: Web Payments
-  section:
-  - title: Intro to the Payment Request API
-    path: /web/fundamentals/payments/
-  - title: Deep Dive into the Payment Request API
-    path: /web/fundamentals/payments/deep-dive-into-payment-request
-  - title: Payment Request UX considerations
-    path: /web/fundamentals/payments/payment-request-ux-considerations
-  - title: Integrating Android Pay into Payment Request
-    path: /web/fundamentals/payments/android-pay
+  - title: Web Payments
```

Comment:
Can you makes sure this uses the same indentation, it looks like this has changed.

Reply:
This should be fixed by now.